src/cdomains/apron/apronDomain.apron.ml

```diff
@@ -515,25 +237,36 @@ sig
   val mem_var : t -> Var.t -> bool
   val assign_var_parallel' :
     t -> Var.t list -> Var.t list -> t
-  val meet_tcons : t -> Tcons1.t -> t
+  val meet_tcons : t -> Tcons1.t -> exp -> t
```

Comment:
What does this `exp` mean?

Reply:
I think this is the original expression to then be able to do the overflow checks for it.